### PR TITLE
FEATURE: ONE-3034 Add new visualizations

### PR DESCRIPTION
**Pull request by pbenes**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/1442/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1190/ : SUCCESS in 1m 13s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/663/ : SUCCESS in 2m 09s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/1446/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1193/ : SUCCESS in 1m 14s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/666/ : SUCCESS in 2m 12s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/1462/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1203/ : SUCCESS in 1m 22s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/676/ : SUCCESS in 2m 15s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/1468/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1209/ : SUCCESS in 1m 53s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/682/ : SUCCESS in 3m 14s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/1473/ : SUCCESS in 16s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1213/ : FAILURE in 1m 25s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/686/ : SUCCESS in 2m 11s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/1478/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1217/ : SUCCESS in 1m 37s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/690/ : SUCCESS in 2m 34s

**Comment by pbenes** [comment]
extended test - storybook

**Comment by pbenes** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/1484/ : SUCCESS in 15s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1221/ : SUCCESS in 1m 29s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/694/ : SUCCESS in 2m 42s

**Comment by osklive-kacatko** [comment]
Link to Storybook test result:

https://ci.intgdc.com/job/gooddata-react-components-storybook-zuul-docker/501/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/1485/ : SUCCESS in 15s
- https://ci.intgdc.com/job/gooddata-react-components-pull-request-dispatcher/429/ : UNSTABLE in 9m 21s

**Comment by pbenes** [comment]
extended test - storybook

**Comment by osklive-kacatko** [comment]
Link to Storybook test result:

https://ci.intgdc.com/job/gooddata-react-components-storybook-zuul-docker/503/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/1488/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-react-components-pull-request-dispatcher/430/ : UNSTABLE in 14m 23s

**Comment by pbenes** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/1495/ : SUCCESS in 19s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1228/ : SUCCESS in 4m 17s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/701/ : SUCCESS in 2m 37s

**Comment by gdgate** [comment]
Merge Failed.

This change or one of its cross-repo dependencies was unable to be automatically merged with the current state of its repository. Please rebase the change and upload a new patchset.

**Comment by pbenes** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/1502/ : SUCCESS in 41s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1234/ : SUCCESS in 1m 46s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/707/ : SUCCESS in 4m 23s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/1504/ : SUCCESS in 24s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1235/ : SUCCESS in 2m 59s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/708/ : SUCCESS in 2m 33s

**Comment by osklive-kacatko** [comment]
Link to Storybook test result:

https://ci.intgdc.com/job/gooddata-react-components-storybook-zuul-docker/511/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/1505/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-react-components-pull-request-dispatcher/437/ : UNSTABLE in 11m 51s

**Comment by pbenes** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/1506/ : SUCCESS in 11s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1236/ : SUCCESS in 3m 01s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/709/ : SUCCESS in 2m 18s

**Comment by osklive-kacatko** [comment]
Link to Storybook test result:

https://ci.intgdc.com/job/gooddata-react-components-storybook-zuul-docker/512/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/1507/ : SUCCESS in 12s
- https://ci.intgdc.com/job/gooddata-react-components-pull-request-dispatcher/438/ : UNSTABLE in 12m 48s

**Comment by pbenes** [comment]
extended test - storybook

**Comment by pbenes** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/1512/ : SUCCESS in 13s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1241/ : SUCCESS in 3m 23s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/714/ : SUCCESS in 2m 22s

**Comment by osklive-kacatko** [comment]
Link to Storybook test result:

https://ci.intgdc.com/job/gooddata-react-components-storybook-zuul-docker/515/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/1513/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-react-components-pull-request-dispatcher/440/ : UNSTABLE in 10m 19s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/1519/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1246/ : SUCCESS in 1m 41s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/719/ : SUCCESS in 2m 32s

**Comment by pbenes** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/1529/ : SUCCESS in 10s
- https://ci.intgdc.com/job/gooddata-react-components-pull-request-dispatcher/445/ : FAILURE in 1m 14s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/1532/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1254/ : FAILURE in 3m 00s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/727/ : SUCCESS in 3m 23s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/1534/ : SUCCESS in 12s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1256/ : FAILURE in 2m 47s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/729/ : SUCCESS in 2m 21s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/1535/ : SUCCESS in 9s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1257/ : SUCCESS in 1m 25s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/730/ : SUCCESS in 2m 23s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/1537/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1259/ : SUCCESS in 3m 14s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/732/ : SUCCESS in 2m 47s

**Comment by pbenes** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://ci.intgdc.com/job/gooddata-react-components-opensource-repository-check/1538/ : SUCCESS in 10s
- https://ci.intgdc.com/job/gooddata-react-components-pull-request-dispatcher/447/ : FAILURE in 16s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/1/ : Tests on opensource repositories should be triggered by users in `gooddata` organization in 7s
- gooddata-react-components-unit-tests-zuul-docker-el7 : SKIPPED
- gooddata-react-components-examples-validation-zuul-docker-el7 : SKIPPED

**Comment by pbenes** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/2/ : Tests on opensource repositories should be triggered by users in `gooddata` organization in 8s
- gooddata-react-components-pull-request-dispatcher : SKIPPED

**Comment by ivan-nejezchleb** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/3/ : Tests on opensource repositories should be triggered by users in `gooddata` organization in 11s
- gooddata-react-components-pull-request-dispatcher : SKIPPED

**Comment by langhammerp** [comment]
extended test - storybook

**Comment by osklive-kacatko** [comment]
Link to Storybook test result:

https://ci.intgdc.com/job/gooddata-react-components-storybook-zuul-docker/522/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/4/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-react-components-pull-request-dispatcher/456/ : UNSTABLE in 11m 40s

**Comment by pbenes** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/5/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1263/ : SUCCESS in 1m 37s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/736/ : SUCCESS in 2m 18s

**Comment by osklive-kacatko** [comment]
Link to Storybook test result:

https://ci.intgdc.com/job/gooddata-react-components-storybook-zuul-docker/523/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/6/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-react-components-pull-request-dispatcher/457/ : SUCCESS in 10m 26s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/10/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1270/ : SUCCESS in 3m 18s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/743/ : SUCCESS in 2m 42s

File: src/components/HeatMap.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+}
+
+type IHeatMapNonBucketProps = Subtract<IHeatMapProps, IHeatMapBucketProps>;
+
+export interface IHeatMapProps extends ICommonChartProps {
+    projectId: string;
+}
+
+function getStackingResultSpec(buckets: VisualizationObject.IBucket[]): AFM.IResultSpec {
+    if (isStackedChart(buckets)) {
```
Comment (pbenes) [inline src/components/HeatMap.tsx]: Do we need to distinguish stacked/non-stacked for heatmap?

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/16/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1276/ : SUCCESS in 4m 00s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/749/ : SUCCESS in 2m 23s

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/28/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1289/ : FAILURE in 1m 55s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/762/ : FAILURE in 1m 24s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/29/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1290/ : SUCCESS in 3m 22s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/763/ : SUCCESS in 2m 55s

**Comment by ZdenekVidensky** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/31/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1292/ : FAILURE in 6m 07s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/765/ : SUCCESS in 2m 31s

**Comment by ZdenekVidensky** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/33/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1293/ : SUCCESS in 1m 36s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/766/ : SUCCESS in 5m 20s

**Comment by yenkins** [comment]
Link to Storybook test result:

https://ci.intgdc.com/job/gooddata-react-components-storybook-zuul-docker/531/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/34/ : SUCCESS in 12s
- https://ci.intgdc.com/job/gooddata-react-components-pull-request-dispatcher/460/ : SUCCESS in 11m 58s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/39/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1298/ : SUCCESS in 9m 31s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/771/ : SUCCESS in 5m 13s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/43/ : SUCCESS in 8s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1302/ : SUCCESS in 3m 09s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/775/ : SUCCESS in 4m 21s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/6/ : SUCCESS in 18s
- https://ci.intgdc.com/job/gooddata-react-components-unit-tests-zuul-docker-el7/1310/ : SUCCESS in 4m 13s
- https://ci.intgdc.com/job/gooddata-react-components-storybook-zuul-docker/532/ : SUCCESS in 11m 22s
- https://ci.intgdc.com/job/gooddata-react-components-examples-validation-zuul-docker-el7/783/ : SUCCESS in 2m 55s

**Pull request merged**